Dockerfile

```diff
@@ -8,10 +8,10 @@ COPY stack-config/config.yaml $STACK_ROOT/config.yaml
 COPY stack-config/global-project/stack.yaml $STACK_ROOT/global-project/stack.yaml
 
 RUN apt-get update \
-    && apt-get -y install curl perl tar \
-    && bash -c "curl -sSL https://get.haskellstack.org/ | sh" \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+  && apt-get -y install curl perl tar llvm libnuma-dev \
```

Comment:
What are those needed for?

Reply:
I tested it on my laptop and without llvm the docker image can't be built. I think llvm is used on arm to enhance the code generation.